<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::BookmarkPropertiesDialog</class>
 <widget class="QDialog" name="Otter::BookmarkPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="buddy">
        <cstring>titleLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
       <property name="buddy">
        <cstring>descriptionTextEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>Address:</string>
       </property>
       <property name="buddy">
        <cstring>addressLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="folderLabel">
       <property name="text">
        <string>Folder:</string>
       </property>
       <property name="buddy">
        <cstring>folderComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="folderComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newFolderButton">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="addressLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="titleLineEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QPlainTextEdit" name="descriptionTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="keywordLabel">
       <property name="text">
        <string>Keyword:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="keywordLineEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="visitsLabel">
       <property name="text">
        <string>Visits:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lastVisitLabel">
       <property name="text">
        <string>Last Visit:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="addedLabel">
       <property name="text">
        <string>Created:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="Otter::TextLabelWidget" name="visitsLabelWidget" native="true">
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="Otter::TextLabelWidget" name="lastVisitLabelWidget" native="true">
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="Otter::TextLabelWidget" name="addedLabelWidget" native="true">
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Modified:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="Otter::TextLabelWidget" name="modifiedLabelWidget" native="true">
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::TextLabelWidget</class>
   <extends>QWidget</extends>
   <header>src/ui/TextLabelWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
